app/javascript/crayons/MarkdownToolbar/__tests__/markdownSyntaxFormatters.test.js

```diff
@@ -10,31 +11,55 @@ describe('markdownSyntaxFormatters', () => {
         const textAreaValue = 'one two three';
         const expectedNewTextAreaValue = 'one **two** three';
 
-        const { newTextAreaValue, newCursorStart, newCursorEnd } =
-          coreSyntaxFormatters['bold'].getFormatting({
-            value: textAreaValue,
-            selectionStart: 4,
-            selectionEnd: 7,
-          });
-
-        expect(newTextAreaValue).toEqual(expectedNewTextAreaValue);
-        expect(
-          newTextAreaValue.substring(newCursorStart, newCursorEnd),
-        ).toEqual('two');
+        const {
+          newCursorStart,
+          newCursorEnd,
+          editSelectionStart,
```

Comment:
all of the changes in this file are essentially the same, instead of asserting the `newTextAreaValue`, we check that if we build the new text area value with the given components, we get the same outcome as we did before